scripts/check_chromium_revision.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2025 The WPT Dashboard Project. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import os
+import requests
+from time import time
+from google.cloud import storage
+
+DEFAULT_TIMEOUT = 600.0
+BUCKET_NAME = 'wpt-versions'
+NEW_REVISION_FILE = 'pinned_chromium_revision_NEW'
+OLD_REVISION_FILE = 'pinned_chromium_revision'
+PLATFORM_INFO = [
+    ("Win_x64", "chrome-win.zip"),
+    ("Win", "chrome-win.zip"),
+    ("Linux_x64", "chrome-linux.zip"),
+    ("Mac", "chrome-mac.zip")
+]
+SNAPSHOTS_PATH = "https://storage.googleapis.com/chromium-browser-snapshots/"
+PR_NUMBER = "50375"
+REPO_OWNER = "web-platform-tests"
+REPO_NAME = "wpt"
```

Comment:
Thoughts on making these environment variables? A user shouldn't really need to change the code just to update these variables. Instead, they could quickly deploy the app with updated variables. You could do more of these variables but this set would be a good start. 

[[1](https://12factor.net/config#:~:text=The%20twelve%2Dfactor%20app%20stores%20config%20in%20environment%20variables%20(often%20shortened%20to%20env%20vars%20or%20env))]

Reply:
I can definitely make these environment variable instead 🙂